envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -12,6 +13,7 @@
 from envs.monkey_zoo.blackbox.log_handlers.test_logs_handler import TestLogsHandler
 
 DEFAULT_TIMEOUT_SECONDS = 5*60
+PERFORMANCE_TIMEOUT_SECONDS = 10*60
```

Comment:
Should be moved to performance test since it's performance test specific.

Reply:
Done